src/vue/components/lookup.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="alpheios-lookup__form" v-if="uiController">
+  <div class="alpheios-lookup__form" v-if="settings">
```

Comment:
I honestly don't know why do we have this `v-if` there. I don't remember putting it in. My goal with this change was to eliminate all direct links to a uiController.

If that's for tests only I agree that we should eliminate it and adjust the tests.

Reply:
I think the reason for this check is that we use `settings` in a `created()`:
```
this.instanceContentOptions = this.settings.contentOptions.clone(TempStorageArea)
this.instanceResourceOptions = this.settings.resourceOptions.clone(TempStorageArea)
```
So if `settings` does not exist, the component cannot function. But it's hard for me to imagine when settings would not exist (as they're managed by a UI controller now), and also we have this additional line of defence: `inject: ['app', 'ui', 'l10n', 'settings']`. It will produce an error if settings are not there.

So I think it's safe to remove a `settings` check. What do you think?

Reply:
yes, I'd rather do that. I think it was there for the ease of tests but I would rather use mock objects for test dependencies, and if the component can't function without the dependent object, then the test shouldn't try to make it do so

Reply:
Removed that